requirements.txt

```diff
@@ -0,0 +1,5 @@
+iniconfig==2.0.0
+numpy==1.26.4
```

Comment:
Using python version 3.8.10 (which unfortunately is the default on some machines), the numpy requirement fails. Maybe change this requirement to 1.24.4 to make it more compatible.

Reply:
I've downgraded the dependency versions.